torch/quantization/quantize.py

```diff
@@ -32,32 +33,22 @@ def _propagate_qconfig_helper(module, qconfig_dict, skip_list=None,
     Return:
         None, module is modified inplace with qconfig attached
     """
-    if skip_list is None:
-        skip_list = DEFAULT_SKIP_LIST
-    if type(module) in skip_list:
-        module.qconfig = None
-    if not hasattr(module, 'qconfig'):
-        module.qconfig = qconfig_parent
-        if qconfig_dict:
-            if prefix in qconfig_dict:
-                module.qconfig = qconfig_dict[prefix]
-            elif type(module) in qconfig_dict:
-                module.qconfig = qconfig_dict[type(module)]
-
-    # Don't quantize empty Sequential, empty Sequential is same as
-    # Identity, but we can't put Sequential into skip list because
-    # we also have non-empty Sequential and the qconfig needs to
-    # be propagated to child in that case
     # TODO: Add test
-    if len(module._modules) == 0 and type(module) == nn.Sequential:
-        module.qconfig = None
+    if white_list is None:
```

Comment:
Why not have white_list= DEFAULT_QCONFIG_PROPAGATE_WHITE_LIST as a default?

Reply:
Because if the `DEFAULT_QCONFIG_PROPAGATE_WHITE_LIST` is mutable, that will create problems